boards/beagle/beaglebone_ai64/beaglebone_ai64_j721e_main_r5f0_0_defconfig

```diff
@@ -13,3 +13,6 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 # Enable Console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Enable I2C
+CONFIG_I2C=y
```

Comment:
This shouldn't go in the `defconfig`.

Reply:
I had recently append this
it is with relation with https://github.com/zephyrproject-rtos/zephyr/pull/80113#pullrequestreview-2416421386 comment

Reply:
I think you should add `tests/drivers/i2c/i2c_api/boards/beaglebone_ai64_...overlay` to make it build there.

Reply:
Removed CONFIG_I2C

Reply:
Hello @pdgendt do I need to add the above?, as it won't be able to compile due to RAM overflow until DDR support comes upstream.

Reply:
The `_defconfig` file is reserved for options that a board minimally needs to be able to boot. You should make sure that it's get build/tested in the tests/samples that only depend on I2C.

Reply:
Ahh, Thanks for clarifying it,
I will be closing this conversation then.